src/components/il-hero/README.md

```diff
@@ -21,6 +22,7 @@ A basic hero contain the page title:
 |                                                          |
 |                                                          |
 +----------------------------------------------------------+
+```
 
 ## Changing the background color
```

Comment:
Does changing the background color also change the text, buttons, etc? Should this be a class?